packages/clerk-js/src/v4/UserProfile/MfaPhoneCodePage.tsx

```diff
@@ -0,0 +1,129 @@
+import { PhoneNumberResource } from '@clerk/types';
+import React from 'react';
+
+import { useCoreUser } from '../../ui/contexts';
+import { useWizard, Wizard } from '../common';
+import { Col, Text } from '../customizables';
+import { ArrowBlockButton, useCardState, withCardStateProvider } from '../elements';
+import { getFlagEmojiFromCountryIso, handleError, parsePhoneString, stringToFormattedPhoneString } from '../utils';
+import { FormButtonContainer } from './FormButtons';
+import { NavigateToFlowStartButton } from './NavigateToFlowStartButton';
+import { ContentPage } from './Page';
+import { AddPhone, VerifyPhone } from './PhonePage';
+import { SuccessPage } from './SuccessPage';
+import { AddBlockButton } from './UserProfileBlockButtons';
+
+export const MfaPhoneCodePage = withCardStateProvider(() => {
+  const title = 'Add multifactor authentication';
+
+  const ref = React.useRef<PhoneNumberResource | undefined>();
```

Comment:
🙃 
`const ref = React.useRef<PhoneNumberResource>();` is the same without using `| undefined` explicitly

Reply:
Fixing